newsfragments/41039.feature.rst

```diff
@@ -0,0 +1 @@
+Enable ``get_current_context()`` to work in virtual environments with some limitations.
```

Comment:
This raises the questuon what limitations.
If its segnificant then we should provide some details, if its not segnificant then we shouldn't mention it here at all (providing the full explnation in the docs)

Reply:
I combined them into a single line.